### ISSUE TITLE
limitlessled-kodi

### ISSUE THREAD

**Issue by jacaba1** [abeeson/limitlessled-kodi#4]
I read the new readme for limitlessled-kodi, but i have Kodi for Windows and i don´t know  to install  the script because i think  the instruccions are for Linux, is correct?

**Comment by abeeson** [comment]
hey mate, sorry about that i should have asked which it was.

It should still work on windows, but i haven't actually tested it. Everything it does is python which should be already supplied and installed by Kodi, but your profile locations will be different.

For Windows, anywhere that says ~/.kodi, swap to %APPDATA\Kodi\ (note the slash changes direction on windows)

Everything else should be right, if you have any trouble finding either the scripts or userdata folder let me know.

**Comment by jacaba1** [comment]
yesterday, i tried install in this folder, but  not working.

How can i see when the script is active and running?

**Comment by abeeson** [comment]
on windows i'm not actually sure you can directly.

Your best bet is to turn on kodi debugging under the kodi system settings page, then watch the kodi.log under the kodi folder.

But before you do that, make sure you have the autoexec.py in the right place, and then if you want to upload the log to pastebin or something and send me a link i'll have a look and see if i can see what the issue is.

**Comment by abeeson** [comment]
actually after having a bit of a look, could you try moving the playeractions.py and the config ini into the userdata folder rather than the scripts folder and try it again?

Remember to restart Kodi afterwards, then let me know if that does / doesn't work and i'll modify the readme.

Sorry i'd test this myself but i'm away all week for training so i'm not able to, if you don't want to do a bunch of testing i'll be home on the weekend and i can run through some testing and update the instructions for you then?

EDIT: I've had a better look, i'm nearly certain that scripts folder was a legacy instruction that shouldn't be there. Definitely try putting them all into the userdata folder / master profile folder (if you have profiles) and give it another go for me. Sorry about that, risks of writing documentation while you are away and unable to actually test it :)

**Comment by jsloane** [comment]
Yeah, on Windows the files go in: %APPDATA%\Kodi\userdata
I don't think the script writes anything to the kodi log, so that won't help.

**Comment by abeeson** [comment]
The script won't but a failure to run the script should, but I'm pretty sure we have sorted it now, jsloane runs it on Windows so he is my authority on this :)

**Comment by jacaba1** [comment]
Yeahhhm yesterday I got it to wor!!!!!

It does not work very well, everytime I turn on kodi, i have plug off and plug on the wifi Receiver bridge controller box.

**Comment by abeeson** [comment]
Yeah that's a known issue, the wireless on the receiver drops out at random, it's super annoying.

If you have any experience with a soldering iron and have a raspberry pi or are willing to buy one it's fairly easy to fix, but unfortunately it's not a problem on the scripts side so I can't fix it :(

Check the faq for instructions on how to convert it to a wired device using the pi.

Glad it's working though!

**Comment by jacaba1** [comment]
OK, thank.

i would like dim only group 3 and 4.   is it posible?

**Comment by abeeson** [comment]
Hey mate,

Sorry it's taken so long to reply, i completely forgot about this

You can have it just dim groups 3 and 4, you'll have to change the code though to send the correct codes for those groups.

It'll be much easier to set up all your lights into one group if possible, then adjust the script to send those codes as needed.

The key is what on / off is sent, for the groups they are:
        ## RGBW ON HEX Values
        $hexValue = "\x45\x00\x55" if $zone == 1;
        $hexValue = "\x47\x00\x55" if $zone == 2;
        $hexValue = "\x49\x00\x55" if $zone == 3;
        $hexValue = "\x4B\x00\x55" if $zone == 4;
        ## RGBW OFF HEX Values
        $hexValue = "\x46\x00\x55" if $zone == 1;
        $hexValue = "\x48\x00\x55" if $zone == 2;
        $hexValue = "\x4A\x00\x55" if $zone == 3;
        $hexValue = "\x4C\x00\x55" if $zone == 4;
        ## Dual White ON HEX Values
        $hexValue = "\x38\x00\x55" if $zone == 1;
        $hexValue = "\x3D\x00\x55" if $zone == 2;
        $hexValue = "\x37\x00\x55" if $zone == 3;
        $hexValue = "\x32\x00\x55" if $zone == 4;
        ## Dual White OFF HEX Values
        $hexValue = "\x3B\x00\x55" if $zone == 1;
        $hexValue = "\x33\x00\x55" if $zone == 2;
        $hexValue = "\x3A\x00\x55" if $zone == 3;
        $hexValue = "\x36\x00\x55" if $zone == 4;

Insert the ones you need into the on and off section, then you should be right to go.

Getting both group 3 and 4 to go will require you to basically make two calls, which will take a bit of stuffing around to make happen....
